docs/whatsnew/7.0.rst

```diff
@@ -123,6 +124,24 @@ value in ``Unit.to`` and have those arrays not be converted to Numpy arrays::
 
 Note that it is not yet possible to use ``Quantity`` with dask arrays directly.
 
+.. _whatsnew_7_0_faster_io_ascii_guessing:
+
+Faster guessing of formats in ``astropy.io.ascii``
+==================================================
+
+When reading text-based tables with ``astropy.io.ascii`` and without specifying the
+table format, guessing the format should now be faster for large tables as it is now
+based on a subset of the table - by default the first 10000 lines of the file. If
```

Comment:
I think some slight variation of my suggestion for the change log would work here to make a more direct statement about the performance improvement.

Reply:
Done